app/services/slack_service.rb

```diff
@@ -3,6 +3,7 @@ module SlackService
 
   def post_hangout_notification(hangout)
     return unless Features.slack.notifications.enabled
+    return if hangout.hangout_url.blank?
```

Comment:
Any particular reason why this would be better than the original method?

Also I discovered that `hangout.hangout_url?` == `hangout.hangout_url.blank?` so maybe that could be a alternative as well.